tests/frontier/opcodes/test_call_code_gas.py

```diff
@@ -96,26 +108,24 @@ def pre(callee_code, caller_code) -> Dict[str, Account]:  # noqa: D103
 
 
 @pytest.fixture
-def post(caller_gas) -> Dict[str, Account]:  # noqa: D103
+def post(sufficient_gas: bool) -> Dict[str, Account]:  # noqa: D103
     return {
-        to_address(Accounts.caller): Account(
-            storage={0x00: (0x01 if caller_gas >= 0x3D50 else 0x00)}
-        ),
+        to_address(Accounts.caller): Account(storage={0x00: 0x01 if sufficient_gas else 0x00}),
     }
 
 
 @pytest.mark.parametrize(
-    "caller_gas",
-    [0x2D5F, 0x3D50],
-    ids=["insufficient", "sufficient"],
-)
-@pytest.mark.parametrize(
-    "caller_type",
-    [Op.CALL, Op.CALLCODE],
+    "caller_type, caller_gas, sufficient_gas",
+    [
+        (Op.CALL, CALL_INSUFFICIENT_GAS, False),
+        (Op.CALL, CALL_SUFFICIENT_GAS, True),
```

Comment:
If it is necessary to really tighten down these gas boundaries, one could also use `CALL_SUFFICIENT_GAS` (after renaming, something like `CALL_SUFFICIENT_GAS_PLUS_INIT_STACK), then the failing test could be sending `CALL_SUFFICIENT_GAS - 1`